docs/source/user/settings.rst

```diff
@@ -314,6 +314,32 @@ the configuration should be simply changed to:
         'default': 'https://login.wifiservice.com/{organization}/password/reset/confirm/{uid}/{token}'
     }
 
+
+``OPENWISP_RADIUS_OPTIONAL_REGISTRATION_FIELDS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``{'first_name': 'disabled', 'last_name': 'disabled', 'location': 'disabled', 'birth_date': 'disabled'}``
+
+This setting is used to specify if the user's optional fields (``first_name``, ``last_name``, ``location`` and ``birth_date``) 
```

Comment:
please explain that this is just the global setting and that it can be overridden at organization level in the admin (add a screenshot) of the fields you added